src/components/ui/avatar/Avatar.mdx

```diff
@@ -0,0 +1,114 @@
+# Avatar
+
+import { Controls, Canvas, Meta, Story, Source } from '@storybook/blocks';
+import * as AvatarStories from './Avatar.stories';
+
+<Meta of={AvatarStories} />
+
+- [Overview](#overview)
+- [Props](#props)
+- [Usage](#usage)
+- [Examples](#examples)
+
+## Overview
+
+A graphical representation of the user, often used in profile sections.
+
+<Canvas
+  of={AvatarStories.Overview}
+  source={{
+    format: true,
+    code: `<Avatar src="https://i.pravatar.cc/300" name="John Doe" />`,
+  }}
+/>
+
+## Props
+
+<Controls />
+
+## Usage
+
+A `Avatar` consists of 3 properties:
+- `size`
+- `src`
+- `name`
+
+The following code shows how you can use the `Avatar` component:
+
+<Source
+  format={true}
+  code={`
+    import { Avatar } from '@/components/ui';
+
+    <Avatar size="sm" src="https://i.pravatar.cc/300" name="John Doe" />
+  `}
+/>
+
+## Examples
+
+### Sizes
+
+Using the `size` prop to change the avatar's size. There are 6 sizes: `xs`, `sm`, `md` (default), `lg`, `xl`, `2xl`.
```

Comment:
The default value for the `size` prop is `md`. You can use the size prop to modify the avatar's dimensions, with six available options: `xs`, `sm`, `md`, `lg`, `xl`, and `2xl`.